proto/provenance/exchange/v1/query.proto

```diff
@@ -68,8 +68,9 @@ service Query {
   }
 
   // GetAccountCommitments gets all the funds in an account that are committed to any market.
+  // Optionally, you can filter the results by a specific asset denomination using the `asset` query parameter.
   rpc GetAccountCommitments(QueryGetAccountCommitmentsRequest) returns (QueryGetAccountCommitmentsResponse) {
-    option (google.api.http).get = "/provenance/exchange/v1/commitments/account/{account}";
+    option (google.api.http).get = "/provenance/exchange/v1/commitments/account/{account}?asset={asset}";
```

Comment:
You don't want the `?asset={asset}` part here. The client/server stuff automatically adds query parameters for stuff that doesn't have a placeholder in the string.

In other words, if the path is `"/provenance/exchange/v1/commitments/account/{account}"`, it'll still work just fine. If there isn't an `asset` value limiting the query, the client won't include a query parameter for it; then, in the server, it'll leave the value as an empty string in `QueryGetAccountCommitmentsRequest `. If there **is* an asset, the client will notice that there isn't a placeholder for it, and add it to the URL as a query parameter which the server then sees and sets in the request.